cmd/influxd/run/server_test.go

```diff
@@ -1394,6 +1394,11 @@ func TestServer_Query_Common(t *testing.T) {
 			exp:     fmt.Sprintf(`{"results":[{"series":[{"name":"cpu","columns":["time","value"],"values":[["%s",1]]}]}]}`, now.Format(time.RFC3339Nano)),
 		},
 		&Query{
+			name:    "explicitly selecting time and a valid measurement and field should succeed",
+			command: `SELECT time,value FROM db0.rp0.cpu`,
```

Comment:
Well, well, that test case fails. The following data comes back:

```
actual: {"results":[{"series":[{"name":"cpu","columns":["value","time"],"values":[["2015-08-20T23:57:35.973310065Z",1]]}]}]}
```

I'd like to address this separately as it will require a somewhat wider set of changes. OK? https://github.com/influxdb/influxdb/issues/3779

I'd like to get this panic fixed before the 0.9.3 window closes.